R/deployApp.R

```diff
@@ -186,10 +185,8 @@ deployApp <- function(appDir = getwd(),
   }
 
   # normalize appDir path and ensure it exists
-  appDir <- normalizePath(appDir, mustWork = FALSE)
-  if (!file.exists(appDir)) {
-    stop(appDir, " does not exist")
-  }
+  check_directory(appDir)
```

Comment:
Pre-existing nit, but this should be merged with the earlier `check_string(appDir)`. By this time, it has already been used in `runStartupScripts`.